packages/@react-spectrum/s2/intl/en-US.json

```diff
@@ -17,6 +17,7 @@
   "label.(required)": "(required)",
   "label.(optional)": "(optional)",
   "menu.moreActions": "More actions",
+  "notificationbadge.plus": "+",
```

Comment:
you'll need to do something like the line below for `tag.showAllButtonLabel`
you should be able to use https://formatjs.github.io/docs/intl#formatmessage as a reference for the API

But the end goal should be that a string comes out of this with a number and the plus sign integrated together.